Card/src/main/java/com/braintreepayments/api/card/BinData.kt

```diff
@@ -0,0 +1,85 @@
+package com.braintreepayments.api.card
+
+import android.os.Parcelable
+import androidx.annotation.RestrictTo
+import com.braintreepayments.api.sharedutils.Json
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+/**
+ * A class to contain BIN data for the card number
+ *
+ * @property prepaid Whether the card is a prepaid card.
+ * @property healthcare Whether the card is a healthcare card.
+ * @property debit Whether the card is a debit card.
+ * @property durbinRegulated A value indicating whether the issuing bank's card range is regulated
+ * by the Durbin Amendment due to the bank's assets.
+ * @property commercial Whether the card type is a commercial card and is capable of processing
+ * Level 2 transactions.
+ * @property payroll Whether the card is a payroll card.
+ * @property issuingBank The bank that issued the credit card.
+ * @property countryOfIssuance The country that issued the credit card.
+ * @property productId The code for the product type of the card
+ * (e.g. `D` (Visa Signature Preferred), `G` (Visa Business)).
+ */
+@Parcelize
+data class BinData(
+    val prepaid: BinType = BinType.Unknown,
+    val healthcare: BinType = BinType.Unknown,
+    val debit: BinType = BinType.Unknown,
+    val durbinRegulated: BinType = BinType.Unknown,
+    val commercial: BinType = BinType.Unknown,
+    val payroll: BinType = BinType.Unknown,
+    val issuingBank: String = BinType.Unknown.name,
+    val countryOfIssuance: String = BinType.Unknown.name,
+    val productId: String = BinType.Unknown.name
+) : Parcelable {
+
+    enum class BinType {
```

Comment:
Should we move this enum class to a separate file? We had followed this pattern in other modules.